src/screens/OperationDetails/Content.js

```diff
@@ -88,6 +89,8 @@ class Content extends PureComponent<Props, State> {
   render() {
     const { account, parentAccount, operation, currencySettings } = this.props;
     const mainAccount = getMainAccount(account, parentAccount);
+    const isToken =
+      listTokenTypesForCryptoCurrency(mainAccount.currency).length > 0;
```

Comment:
I'm getting confused... isn't what the UI call "subaccount" really means the `ChildAccount` type?

Wouldn't that break in case we integrate a currency supporting both tokens and subaccounts/childaccounts?

Isn't it possible to just check for `account.type === "TokenAccount"` (or we have some accounts typed as `TokenAccount` which should be displayed as _subaccounts_?)

Reply:
I mimicked the logic from lldv2 https://github.com/ledgerhq/lld-v2/blob/820fd4e3d4f1ecab1bbfd16a5d3216faac350e1c/src/renderer/modals/OperationDetails/index.js#L327 which ironically comes from you 🤣

Reply:
I'm not really the author of that line, I just did the v1 to v2 copypasta :trollface: 

@valpinkman I'm probably missing something, but I don't get why you did `listTokenTypesForCryptoCurrency(mainAccount.currency).length > 0` rather than `account.type === "TokenAccount"` to check if an account is a `Token` in
https://github.com/LedgerHQ/ledger-live-desktop/blob/master/src/components/modals/OperationDetails.js#L243

I this some kind of pattern we should continue to follow?
Shouldn't `account.type` always be trusted? 😕

Reply:
oh sorry for that, I trusted the git _blame_

Reply:
My bad, I thought the check was at the child level rather than on the parent,  don't mind me, I'm going burry myself.